src/components/Subscribe/_subscribe.scss

```diff
@@ -4,34 +4,57 @@
 @import '#scss/typography';
 
 .#{$px}-subscribe {
-  max-width: 30rem;
+  max-width: 600px;
   width: 100%;
 
+  .#{$px}-input {
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
+    }
+
+    &__validation {
+      padding-left: 0;
+    }
+  }
```

Comment:
This should probably be applied to the input component itself. I see that the padding got broken on that input at some point,